BuildForSDG/team-029-frontend
# feat: add app dashboard structure

## Description
- add application dashboard using figma design guide
- update App component to include dashboard routes
- use mocked routest on dashboard
- add sidebar component for dashboard
- update app to use withNavbar component for pages utilizing navbar

Fixes #20

## How Has This Been Tested?
- Unit test in place for all added feature
- PR includes a screenshot for the task done
- clone this repository
- install dependencies with `npm install`
- run dev server using `npm run start`
- you should see the app running on port `localhost:3000`
- move to `/dashboard` 
- you can now view the added feature

## Screenshots (if applicable, else remove this line / section)

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have added necessary inline code documentation
- [ x] I have added tests that prove my fix is effective and that this feature works
- [ x] New and existing unit tests pass locally with my changes
![Screenshot from 2020-05-25 03-05-37](https://user-images.githubusercontent.com/32667315/82862703-3d9dba80-9f18-11ea-8732-b6447eef3b38.png)
